node-red-contrib-bot-message/bot-send-card.js

```diff
@@ -538,89 +546,76 @@ const addImageToAdaptiveCard = (imageUrl, imageSize, imageHorizontalAlignment, i
     }
 };
 
-const buildReplyAdaptiveCard = (RED, node, locale, data, config, reply) => {
-    let title = config.titleAdaptiveCard;
-    let subtitle = config.subtitle;
-    let attach = config.attachAdaptiveCard;
-    let attachSize = config.attachSizeAdaptiveCard;
-    let imageHorizontalAlignment = config.attachHorizontalAlignment;
-    let imageWidth = config.attachWidthAdaptiveCard;
-    let imageHeight = config.attachHeightAdaptiveCard;
-    let subtext = config.textAdaptiveCard;
-    let separator = config.containerSeparatorAdaptiveCard;
-    let displayedTextSize = config.displayedTextSizeAdaptiveCard;
-    let titleShowCardAction = config.titleShowCardAction;
-    let textButtonMarker = config.textButtonMarker;
-    let textButtonPrefix = config.textButtonPrefix;
-    
+const buildReplyformAdaptiveCard = (RED, node, locale, data, config, reply) => {
+    let formtitle = config.titleformAdaptiveCard;
+    let errorMsgform = config.errorMsgformAdaptiveCard;
+    let submitLabelform = config.submitLabelformAdaptiveCard;
+    let placeHolderform = config.placeHolderformAdaptiveCard;
+
     if(config.sendType === 'formAdaptiveCard'){
-        title = helper.getContextValue(RED, node, data, title || '', config.titleTypeAdaptiveCard || 'str');
-        subtext = helper.getContextValue(RED, node, data, subtext || '', config.node-input-subtext || 'str');
-        subtext = marshall(locale, subtext,  data, '');
-        title = marshall(locale, title,  data, '');
-        
-        
-        reply.type = 'AdaptiveCard';
-        reply.title = title;
-        reply.version = "1.3";
-        reply.body = [{"type": "TextBlock",
-        "size": "Medium",
-        "weight": "Bolder",
-        "text": title,
-        "horizontalAlignment": "Center",
-        "wrap": true},
-        {
-            "type": "TextBlock",
-            "text": subtext,
-            "size": "Large",
-            "weight": "Bolder",
-            "wrap": true
-        },
-        {
-            "type": "Input.Text",
-            "style": "text",
-            "isMultiline": true,
-            "id": "userFeedback",
-            "placeholder": "master.placeholder",
-            "maxLength": 500,
-            "isRequired": true,
-            "label": "Feedback:",
-            "errorMessage": "master.errorMessage"
-        }];
-        reply.actions = [{
-            "type": "Action.Submit",
-            "title": "master.submit",
-            "data": {
-                "id": "feedbackButton",
-              }
-        }];
-        const textToShow = marshall(locale, subtext,  data, '');
-    }
+    formtitle = helper.getContextValue(RED, node, data, formtitle || '', config.titleformTypeAdaptiveCard || 'str');
+    formtitle = marshall(locale, formtitle,  data, '');
+    
+    
+    reply.type = 'AdaptiveCard';
+    reply.formtitle = formtitle;
```

Comment:
line 561 to be removed, because there is no property 'formtitle' in the MS AdaptiveCard's schema